### DISCUSSION TITLE
/repair charge

### DISCUSSION THREAD

**Discussion by 7hatGuyAJ** [Zrips/CMI#7182] [Enhancement]
Add the ability to change the currency which is used for the /repair command.

currently we can only use in-game money but I would like the option to use points so I can charge players job points or maybe even voting points rather than in-game money.

**Comment by mrfloris** [comment]
PyroWelcomes Points
JobsReborn Points
mcMMO Levels
Adapt Exp/Levels
Economy money Balance
Gem Items
Game Exp
.. omgosh there's so much. I know the zrips plugin TradeMe has some currencies we can set. would be fun if the CMILib could be the main hub config for these plugins, and then each plugin could use that config to show the type(s) of currencies that are configured. Gosh, that will probably be a huge rewrite.
